can_bus/json/VC/VC_alerts.json

```diff
@@ -0,0 +1,14 @@
+{
+    "warnings_id": 1305,
+    "faults_id": 1306,
+    "warnings_counts_id": 1307,
+    "faults_counts_id": 1308,
+    "warnings": [
+        "WatchdogTimeout",
+        "TxOverflow",
+        "RxOverflow"
+    ],
+    "faults": [
+        "DummyFault"
+    ]
```

Comment:
I believe we want to make Quadruna JSONCAN its own separate JSONCAN subproject in its own subfolder, since otherwise there will be lots of conflicts with boards like BMS and FSM. also it means that when we're done with Thruna we can just delete its JSON without affecting Quadruna.

So the structure would look something like:
```md
can_bus/
    dbcs/
    thruna/
        BMS/
        ...
    quadruna/
        VC/
        ...
```

Reply:
also right now we reserve a group of 100 CAN IDs per board. so for example the inverters use 0-99, BMS uses 100-199, DCM uses 200-299, etc. In CAN, lower message means higher priority, so the boards are rougly ordered in "importance".

I think we should probably stick with this for Quadruna for now, if only for organization, so put the VC in the 200-299 range. (0-99 are still inverters, 100-199 is still BMS, and then VC).